dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/AsyncEnrichedDexApi.scala

```diff
@@ -199,6 +199,12 @@ class AsyncEnrichedDexApi(apiKey: String, host: => InetSocketAddress)(implicit e
       .followRedirects(true)
   }
 
+  override def orderBookInfo(amount: String, price: String): R[HttpOrderBookInfo] = mk {
+    sttp
+      .get(uri"$apiUri/matcher/orderbook/$amount/$price}/info")
+      .followRedirects(false)
+  }
+
   override def orderBookInfo(assetPair: AssetPair): R[HttpOrderBookInfo] = mk {
```

Comment:
We could use `orderBookInfo(String, String)`:
```scala
override def orderBookInfo(assetPair: AssetPair): R[HttpOrderBookInfo] = 
  orderBookInfo(assetPair.amountAssetStr, assetPair.priceAssetStr)
```

Reply:
ok